Log analytics docs (#17015)
demisto/content: Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/README.md

You need to grant Cortex XSOAR authorization to access Azure Log Analytics.

**Note**: The Azure account must have permission to manage applications in Azure Active Directory (Azure AD). Any of the following Azure AD roles include the required permissions:

- Application administrator
- Application developer
- Cloud application administrator

> And assigned the "Log Anayitics Reader" role to the user that granted the authorization

Reviewer: ```suggestion
In addition, the user needs to be assigned the **Log Analytics Reader** role.
```

> And assigned the "Log Anayitics Reader" role to the user that granted the authorization

Reviewer: if we have the ms docs to refer to, lets add a link here
